docs/google/engine.md

```diff
@@ -191,3 +191,56 @@ for b in range(num_circuits_in_batch):
      idx+=1
 ```
 
+
+## Downloading historical results
+
+Results from  previous computations are archived and can be downloaded later
+by those in the same cloud project.  You must use the same project id to
+access historical results or your request will be denied.
```

Comment:
What do you think of adding a little motivation for the "same project" requirement by explaining that program IDs are unique within a project?

Reply:
Added a bit about this later.  See what you think.

Reply:
LGTM, thanks!